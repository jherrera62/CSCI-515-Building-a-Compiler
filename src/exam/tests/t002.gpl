on akey
{
    print("first");
    print("second");
    print("third");
    print("fourth");
    print("fifth");
    print("sixth");
    print("seventh");
    print("eighth");
    print("ninth");
    print("tenth");
}
