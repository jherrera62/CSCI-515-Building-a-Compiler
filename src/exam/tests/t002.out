gpl.cpp::main()
  input file(tests/t002.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[3]: first
print[4]: second
print[5]: third
print[6]: fourth
print[7]: fifth
print[8]: sixth
print[9]: seventh
print[10]: eighth
print[11]: ninth
print[12]: tenth

User quit program. Printing the symbol table.
